### PR TITLE
editor: delete key support

### PR DESCRIPTION
**Pull request by tvanderstad**
fixes https://github.com/lockbook/lockbook/issues/1611

@steverusso UAT this before merging!

### CODE REVIEW

File: libs/editor/egui_editor/src/events.rs [hunk 1]
```diff
@@ -211,7 +211,7 @@ fn calc_modification(
 
                 cursor.selection_origin = None;
             }
-            Event::Key { key: Key::Backspace, pressed: true, modifiers } => {
+            Event::Key { key: Key::Backspace | Key::Delete, pressed: true, modifiers } => {
```
[the remark this answers is not in the document]
Comment (tvanderstad) [reply libs/editor/egui_editor/src/events.rs:214]: I've been using a mac for too long 😅

**Comment by steverusso** [comment]
Regular delete works (removes a single character after the cursor), but using delete with selected text just does a regular delete (removes one character after the end of the selection) and then de-selects the selected text.

**Comment by Parth** [comment]
> Regular delete works (removes a single character after the cursor), but using delete with selected text just does a regular delete (removes one character after the end of the selection) and then de-selects the selected text.

potentially related: #1665

**Review by Parth — APPROVED** [review]

**Comment by Parth** [comment]
regular delete & deleting with selection works

**Pull request merged**